config/default/imagemagick.settings.yml

```diff
@@ -1,6 +1,6 @@
 _core:
   default_config_hash: cp-t_XKrU3l5dl9BIOiM-favydmIDTQhL11eF-1MjRc
-quality: 85
+quality: 80
```

Comment:
Note: These compression changes (although minor) contribute to the performance gains achieved in the linked performance tests. These were ported over from a previously rejected PR.